### PR TITLE
Release v2.9.1

### PR DESCRIPTION
**Pull request by rishi-104276**
## Description
This PR contains updated CLI version fixing the issues we are going to release in v2.9.1

## Jira
https://uipath.atlassian.net/browse/TS-1787
https://uipath.atlassian.net/browse/TS-1788
https://uipath.atlassian.net/browse/TS-1824
https://uipath.atlassian.net/browse/TS-1798
https://uipath.atlassian.net/browse/TS-1802

### CODE REVIEW

**Review by ganeshborle — APPROVED** [review]
LGTM

File: src/main/java/com/uipath/uipathpackage/UiPathTest.java [hunk 1]
```diff
@@ -301,8 +306,22 @@ public TraceLevel getTraceLevel() {
...
+		return attachRobotLogs;
+	}
 
-    private void validateParameters() throws AbortException {
+    @DataBoundSetter
+    public void setAttachRobotLogs(boolean attachRobotLogs) {
+    	this.attachRobotLogs = attachRobotLogs;
+    }
+    
+	private void validateParameters() throws AbortException {
```
Comment (amoluipath) [inline src/main/java/com/uipath/uipathpackage/UiPathTest.java:324]: Fix indentation.

File: src/main/java/com/uipath/uipathpackage/UiPathTest.java [hunk 2]
```diff
@@ -301,8 +306,22 @@ public TraceLevel getTraceLevel() {
...
+     *
+     * @return boolean attachRobotLogs
+     */
+    public boolean getAttachRobotLogs() {
+		return attachRobotLogs;
+	}
 
-    private void validateParameters() throws AbortException {
+    @DataBoundSetter
+    public void setAttachRobotLogs(boolean attachRobotLogs) {
```
Comment (amoluipath) [inline src/main/java/com/uipath/uipathpackage/UiPathTest.java:320]: Why does the getter above have a documentation header and why do we not have the documentation header for this setter?

File: src/main/java/com/uipath/uipathpackage/models/TestOptions.java [hunk 3]
```diff
@@ -64,4 +65,13 @@ public String getParametersFilePath() {
...
     }
+
+	public boolean getAttachRobotLogs() {
+		return attachRobotLogs;
+	}
+
+	public void setAttachRobotLogs(boolean attachRobotLogs) {
+		this.attachRobotLogs = attachRobotLogs;
+	}
+
```
Comment (amoluipath) [inline src/main/java/com/uipath/uipathpackage/models/TestOptions.java:76]: Fix indentation.

File: src/main/java/com/uipath/uipathpackage/UiPathTest.java (hunk 2 above)
Comment (rishi-104276) [reply src/main/java/com/uipath/uipathpackage/UiPathTest.java:325]: done

File: src/main/java/com/uipath/uipathpackage/models/TestOptions.java (hunk 3 above)
Comment (rishi-104276) [reply src/main/java/com/uipath/uipathpackage/models/TestOptions.java:76]: done

File: src/main/java/com/uipath/uipathpackage/UiPathTest.java (hunk 1 above)
Comment (rishi-104276) [reply src/main/java/com/uipath/uipathpackage/UiPathTest.java:329]: done

**Pull request merged**